frontend/src/metabase/databases/components/DatabaseInfoField/DatabaseInfoField.tsx

```diff
@@ -7,7 +7,11 @@ export interface DatabaseInfoFieldProps {
 const DatabaseInfoField = ({
   placeholder,
 }: DatabaseInfoFieldProps): JSX.Element => {
-  return <InfoBanner>{placeholder}</InfoBanner>;
+  return (
```

Comment:
I figured out how to render this following #30330 with a little bit of tinkering by hard code `cloud-gateway-ips` to `"1.2.3.4,5.6.7.8"`

The border radius doesn't work
![image](https://github.com/user-attachments/assets/64f0df82-3647-4373-826a-f9004d14dad2)

you should pass `rem` unit instead I think

Reply:
Thinking about this, it's actually quite silly that we use Banner to render this!

Reply:
@WiNloSt thanks for digging into this. We were using the `Banner` in a one-off fashion here.
I'm pretty sure we have countless other ways of presenting this same type of component. I checked in our Storybook and it hasn't been defined there.

To avoid further coupling, I removed the dependency on the `Banner` in https://github.com/metabase/metabase/pull/49273/commits/76cf0bf4a005f5dd2ca4c8f219225261b65a78bb

Will follow up with our designers to see if they wish to formalize this UI pattern.

Reply:
![image](https://github.com/user-attachments/assets/6c38c4a8-ce0d-4cf2-ac91-97904d5458fa)

Reply:
that's awesome